Frontend/ui-customer/src/Services/API.purs

```diff
@@ -17,7 +17,7 @@ module Services.API where
 
 import Control.Alt ((<|>))
 import Control.Monad.Except (runExcept)
-import Common.Types.App (Version(..))
+import Common.Types.App (Version(..), FeedbackAnswer)
```

Comment:
Can you remove this unused import - `FeedbackAnswer`

Reply:
no we are using this in `RideFeedbackReq`